File: VB/ReplaceQueryBuilderSample/FrmDesigner.vb
Imports DevExpress.DataAccess.UI.Sql
Imports DevExpress.XtraEditors
Imports DevExpress.XtraReports.Wizards3
' ...

Namespace ReplaceQueryBuilderSample
    Partial Public Class FrmDesigner
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
            Dim customizationService As New MyWizardCustomizationService()
            reportDesignerMDIController.AddService(GetType(IWizardCustomizationService), customizationService)
            reportDesignerMDIController.AddService(GetType(ISqlEditorsCustomizationService), customizationService)
            reportDesignerMDIController.CreateNewReportWizard()
        End Sub
    End Class
End Namespace

File: CS/ReplaceQueryBuilderSample/FrmDesigner.cs
using DevExpress.DataAccess.UI.Sql;
using DevExpress.XtraEditors;
using DevExpress.XtraReports.Wizards;
// ...

namespace ReplaceQueryBuilderSample {
    public partial class FrmDesigner : XtraForm {
        public FrmDesigner() {
            InitializeComponent();
            MyWizardCustomizationService customizationService = new MyWizardCustomizationService();
            reportDesignerMDIController.AddService(typeof(IWizardCustomizationService), customizationService);
            reportDesignerMDIController.AddService(typeof(ISqlEditorsCustomizationService), customizationService);
            reportDesignerMDIController.CreateNewReportWizard();
        }
    }
}
